### PR TITLE
Stop task execution reference documentation.

### LINKED ISSUE #3345: Document Task Execution Stop
Provide reference documentation for task execution stop

### PR DESCRIPTION
**Pull request by cppwfs**
resolves #3345

### CODE REVIEW

File: spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc [hunk 1]
```diff
@@ -319,6 +319,13 @@ On this screen the user can view not only the information from the Task Executio
 * Task Execution Duration
 * Task Execution Exit Message
 
+==== Stop Executing Tasks
+
+To submit a stop task execution request to the platform, click the drop down button next to the task execution that needs to be stopped.
+Now click the `Stop task` option.   The  dashboard will the present a dialog box asking if you are sure that you want to stop the task execution, click `Stop Task Execution(s)`.
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/dashboard.adoc]: It seems like extra spaces between the words `   The  dashboard`

File: spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc [hunk 2]
```diff
@@ -351,6 +351,40 @@ bad-timestamp is an invalid task.
 
 In this case Spring Cloud Data Flow states that the task is invalid because task:badtimestamp has an invalid URI.
 
+[[spring-cloud-dataflow-stopping-task-execution]]
+=== Stopping a Task Execution
+
+In some cases a task that is executing on a platform may not stop because of a problem on the platform or the software itself.
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: maybe:

> ... or the application business logic itself.

File: spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc [hunk 3]
```diff
@@ -351,6 +351,40 @@ bad-timestamp is an invalid task.
...
+To do this a user can submit a `task execution stop` for a given set of task executions.  For example:
+
+[source,bash]
+----
+task execution stop --ids 5
+
+Request to stop the task execution with id(s): 5 has been submitted
+----
+In the example above we see that the user has requested that task execution id 5 to be stopped.
+In these cases the platform will stop the execution.  When we view the result for the task execution we will see that the task execution completed with a 0 exit code.
```
Comment (sabbyanandan) [inline spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: Perhaps:

> With the above command, the trigger to stop the execution of `id=5` will be submitted to the underlying deployer implementation, and as a result, the operation will stop the execution of that task. When we view the result for the task execution, we will see that the task execution completed with a 0 exit code.

File: spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc (hunk 2 above)
Comment (cppwfs) [reply spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: 👍  done

File: spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc (hunk 3 above)
Comment (cppwfs) [reply spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: 👍 done

File: spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc [hunk 4]
```diff
@@ -351,6 +351,39 @@ bad-timestamp is an invalid task.
...
+║timestamp │1 │Mon Jul 15 09:26:41 EDT 2019│Mon Jul 15 09:26:41 EDT 2019│0        ║
+╚══════════╧══╧════════════════════════════╧════════════════════════════╧═════════╝
+----
+
+[WARNING]
+====
+When stopping a task execution that has a running Spring Batch job, the job will be left with the batch status of `STARTED`.
+====
+
+If a user submits a stop for a task execution that has child task executions associated with it, like a composed task, a stop request will be sent for each of the child task executions.
```
Comment (ilayaperumalg) [inline spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: Can we elaborate on why the status is `STARTED` as the Batch job is expected to go into `STOPPING` -> `STOPPED` status?

Comment (cppwfs) [reply spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: Added additional content on this topic.

Comment (ilayaperumalg) [reply spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc]: Thank you @cppwfs! LGTM, merging.

**Pull request merged**